### PR TITLE
basic components for filters

### PR DESCRIPTION
**Pull request by GraysonQuartech**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,7 +1,12 @@
 import React from "react";
+import FilterRow from "./components/FilterRow";
 
 function App() {
```
Comment (MacQSL) [inline src/App.tsx:4]: Just remember all functions need return types

File: src/components/Filter.tsx [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import { Select } from "@mui/material";
+
+const myArray: string[] = ["apple", "banana", "cherry"];
```
Comment (MacQSL) [inline src/components/Filter.tsx:4]: I know temp stuff, but create an directory called utils.
Add a constants.ts file for stuff like this

Comment (MacQSL) [reply src/components/Filter.tsx:4]: Also this is the directory where you can create a helper_functions.ts file

Comment (GraysonQuartech) [reply src/components/Filter.tsx:4]: done.

**Pull request merged**